reference/constraints/DateTime.rst

```diff
@@ -29,7 +29,7 @@ Basic Usage
         class Author
         {
             /**
-             * @Assert\DateTime
+             * @Assert\Type("\DateTimeInterface")
```

Comment:
This doesn't look right to me. The example here demonstrates how to validate that a string (see the `@var` annotation below) represents a valid date. The `Type` constraint is documented in its own reference document.

Reply:
Ok, @xabbuh I agree with you. I didn't realize that the previous code was checking if the field is well-formatted, not the type of the field. Thank you for your reply.